### ISSUE TITLE
vsce package fails

### ISSUE THREAD

**Issue by hildjj** [peggyjs/code-peggy-language#15]
After #14, `vsce package` fails with:

```
[tsl] ERROR
      TS6059: File '[base]//src/main.ts' is not under 'rootDir' '[base]/server/src'. 'rootDir' is expected to contain all source files.
  The file is in the program because:
    Root file specified for compilation
ts-loader-default_bf8b548751077c90
```

This is why I had removed the tsconfig file in `server/`, but I'm sure there's a way to solve this that is more correct.

**Comment by hildjj** [comment]
@SrTobi can you take a look at this, please?

**Comment by SrTobi** [comment]
Hmm interesting... I tried it on my pc and `vsce package` succeeded and created the package which I install in vscode.

Nevertheless, I made a PR in which I restructured the code a little...

Especially the code structure would be like this:
```
server/server.ts
client/client.ts

out/server/server.js
out/client/client.js
```

- Both `watch:tsc` and webpack will create this out directory.
- With this, no `server.d.ts` is created anymore.
- Debugging works in the client and server.

I also added github actions so that at least package and linting is checked on pull requests
